MediaAdminBundle/Resources/public/template/Media/Modal/mediaModalContainer._tpl.html

```diff
@@ -12,6 +12,9 @@ <h4 class="modal-title"></h4>
                             </option>
                         <% }); %>
                     </select>
+                    <% if (can_create) { %>
+                        <button class="btn btn-info upload-popup-mode"><%- Translator.trans('open_orchestra_backoffice.add')%></button>
```

Comment:
The button should be next to the site selector and i can't is insite container with class styled-select